vault/changelog.md

```diff
@@ -2,10 +2,30 @@
 id: 9bc92432-a24c-492b-b831-4d5378c1692b
 title: Changelog
 desc: ''
-updated: 1666707855927
+updated: 1667834907455
 created: 1601508213606
 nav_order: 2
 ---
+## 0.117.0
+
+### Enhancements
+
+- enhance(workspace): support adding existing vault to the workspace (#3655) @joshi ([[docs|dendron://dendron.dendron-site/dendron.ref.commands]])
+- enhance(workspace): Create enableEngineV3 dev config (#3703) @tuling
```

Comment:
remove - this is currently a dev flag

Reply:
also for future note, we want consistent casing here so this shouldn't be capitalized